src/main/java/core/basesyntax/FileWork.java

```diff
@@ -1,7 +1,37 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.Arrays;
+
 public class FileWork {
+    static final String WORD_START_W = "w";
+
     public String[] readFromFile(String fileName) {
-        return null;
+        StringBuilder lines = new StringBuilder();
+
+        try (BufferedReader reader = new BufferedReader(new FileReader(fileName))) {
+            String value = reader.readLine();
+            while (value != null) {
+                lines.append(value.toLowerCase()).append(System.lineSeparator()).append(" ");
```

Comment:
what is the purpose of appending the lineSeparator here?

also, we may make whitespace a constant field as well